utility/mm2_update-master-directory-list

```diff
@@ -76,46 +88,54 @@ def make_file_details_from_checksums(session, diskpath, relativeDName, D):
         d = {}
         checksum_files = []
         for g in globs:
-            checksum_files.extend(
-                glob.glob(os.path.join(diskpath, relativeDName, g)))
+            if files != None:
```

Comment:
If I remember it correctly I was looking for a way to avoid duplicate code when writing this. But I usually try to avoid long python lines like you proposed as I find it harder to read in this case than the duplicate code.